driver-mct/cime_config/config_component_e3sm.xml

```diff
@@ -390,6 +390,7 @@
       <value compset=".+" grid="a%ne1024np4">864</value>
       <value compset=".+" grid="a%ne0np4_arm_x8v3" >144</value>
       <value compset=".+" grid="a%ne0np4_conus_x4v1" >96</value>
+      <value compset=".+" grid="a%ne0np4_arcticx4v1">96</value>
```

Comment:
@apcraig thanks this looks like exactly what we need to set the correct coupling frequency for the `arcticx4v1_ARRM10to60E2r1` configuration